app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: callable) -> callable:
+    dict_cache = {}
+
+    def inner(*args, **kwargs) -> int:
+        key = (args, frozenset(kwargs.items()))
```

Comment:
Why do we freeze kwargs ordering here?

Reply:
> Why do we freeze kwargs ordering here?

Just one way to unpack the kwargs to keep them organized. I can change that to tuple(). It would be better?